### ISSUE TITLE
--file path only works when in base folder of repo

### ISSUE THREAD

**Issue by grahampugh** [gkluoe/git2jss#7]
git2jss fails when calling a full path to a file. You have to stay at the base of the repo for git2jss to find the file.   

This fails:
```
cd /path/to/jss-scripts-repo
cd subfolder
git2jss --no-keychain --file myscript.sh --name "My Script" --tag v0.2
```

This fails:
```
git2jss --no-keychain --file /path/to/jss-scripts-repo/subfolder/myscript.sh --name "My Script" --tag v0.2
```

This works:
```
cd /path/to/jss-scripts-repo
git2jss --no-keychain --file subfolder/myscript.sh --name "My Script" --tag v0.2
```

I guess it's clear that it's only looking in the current directory for git information. But perhaps it could add some intelligence to traverse directories until it finds that information, or else to supply the base directory. This would help when using git2jss in automation.

**Comment by geoffklee** [comment]
Thanks for the feedback! I agree the way it works at the moment is a bit clunky, and I'd like to improve it, but I'm nervous (perhaps for no good reason) about traversing the directory hierarchy to locate the git repo.  
I like the idea of specifying the base directory though - this seems similar to the `-C` option to `tar`, for instance. I'm just back from paternity leave but should have time to look at this soon - or of course a PR would be very welcome.

**Comment by geoffklee** [comment]
@grahampugh in v1.1.0rc1 you can now specify `--local-repo DIRECTORY`, and the path to any file specified with `--all` or `--file` will be assumed to be relative to _that_ directory.

**Comment by geoffklee** [comment]
I've released version 2.0.0 now, which includes this.
